packages/uui-toast-notification-layout/package.json

```diff
@@ -31,10 +31,7 @@
   ],
   "dependencies": {
     "@umbraco-ui/uui-base": "0.0.17",
-    "@umbraco-ui/uui-button": "0.1.10",
-    "@umbraco-ui/uui-css": "0.0.4",
-    "@umbraco-ui/uui-icon": "0.0.5",
-    "@umbraco-ui/uui-icon-registry-essential": "0.0.3"
+    "@umbraco-ui/uui-css": "0.0.4"
```

Comment:
What is imported from uui-css in this package?

Reply:
We use the uui-text class internally, see the styles prop where UUITextStyles is appended and as well in the render method where the uui-text class is used. `<div id="message" class="uui-text">`